tests/fixtures.ts

```diff
@@ -212,6 +234,10 @@ export class PzFixture extends Fixture {
     }
     
     async initialize(): Promise<PzFixture> {
+        /// in case `this.handleName` is updated after constructor
+        (this.oldCip68Datum.constructor_0[0] as any)['name'] = `$${this.handleName}`;
+        (this.newCip68Datum.constructor_0[0] as any)['name'] = `$${this.handleName}`;
```

Comment:
I believe the constructor already does this?

Reply:
This is when we change the handle name after constructor.
For example, when we test `PERSONALIZATION` with `NFT_SUBHANDLE` or `VIRTUAL_HANDLE`, we have to change handle name.

```typescript
fixture.handleName = 'dev@golddy'; /// nft subhandle
```

Reply:
resolved
Moved to certain test itself